charts/jenkins/templates/_helpers.tpl

```diff
@@ -336,7 +336,9 @@ jenkins:
   {{- if .Values.controller.csrf.defaultCrumbIssuer.enabled }}
   crumbIssuer:
     standard:
+      {{- if contains (include "controller.image.tag" .) "lts" }}
```

Comment:
this won't work you'll either need to check for 2 dots in the version or just if it doesn't equal what's in https://github.com/Lerentis/helm-charts/blob/main/charts/jenkins/Chart.yaml#L6

Reply:
had a look at https://hub.docker.com/r/jenkins/jenkins/tags?name=lts&page=2 and it seems like i still need to take the literal "lts" into consideration. this gets rather complex with this tagLabel logic there as well, but i hope i got it now